products.html

```diff
@@ -0,0 +1,229 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Arcade Store</title>
+    <script src="https://kit.fontawesome.com/4a9487b343.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://nifty-tesla-85dda2.netlify.app/Components/components.css">
+    <link rel="stylesheet" href="common.css">
+    <link rel="stylesheet" href="products.css">
+</head>
+
+<body>
+    <nav class="navbar">
+        <div class="nav-logo">
+            <img src="./Assets/nav-logo.png" alt="logo" />
+        </div>
+        <div class="nav-items">
+            <a class="nav-link sm-title hide-md" href="">Shop</a>
+            <a class="nav-link sm-title hide-md" href="">Categories</a>
+        </div>
+        <div class="nav-space"></div>
+        <div class="nav-icon-items">
+            <div class="nav-search-bar hide-sm">
+                <input class="search-bar" type="text" placeholder="Search..">
+                <i class="fa fa-brands fa-searchengin"></i>
+            </div>
+            <button id="dark-bg-icon" class="btn icon-btn hide-md">
+                <i class="fa fa-solid fa-user"></i>
+            </button>
+            <button id="dark-bg-icon" class="btn icon-btn hide-md">
+                <i class="fa fa-solid fa-heart"></i>
+            </button>
+            <button id="dark-bg-icon" class="btn icon-btn hide-md">
+                <i class="fas fa-shopping-cart"></i>
+            </button>
+            <button id="dark-bg-icon" class="btn icon-btn show-md">
+                <i class="fas fa-solid fa-bars"></i>
+            </button>
+        </div>
+    </nav>
+    <div class="products-layout">
+        <aside class="filters-container light-txt">
+            <div class="filter-header">
+                <p class="sm-title">FILTERS</p>
+                <p class="red-txt n-wt xsm-title filter-clear">CLEAR ALL</p>
+            </div>
+            <div>
+                <div class="filter-header">
+                    <p class="sm-sub-title n-wt">PRICE</p> <span class="xsm-p">100 - 1000</span>
+                </div>
+                <input id="range-slider" class="mg-t-20" type="range" min="0" max="100" value="50">
+            </div>
+            <div>
+                <p class="sm-sub-title n-wt">CATEGORY</p>
+                <div class="category-check mg-t-20">
+                    <input type="checkbox" id="gameCD" name="gameCD" value="Bike">
+                    <label for="gameCD">Fandom Collectibles</label>
+                </div>
+                <div class="category-check mg-t-15">
+                    <input type="checkbox" id="gameCD" name="gameCD" value="Bike">
```

Comment:
@Shivani-1524  one thing I would like to point out here is, users can select multiple checkboxes right so here you have used the same id to wrap the label, so you can have a different id for a particular checkbox to wrap with label.

Reply:
i just copy pasted xD as this is just the css screens afterall but will change it :)